src/Agent.Listener/SelfUpdater.cs

```diff
@@ -250,7 +250,7 @@ private async Task DownloadLatestAgent(CancellationToken token)
                     
                     if (string.IsNullOrEmpty(tar))
                     {
-                        throw new NotSupportedException($"tar -xzf");
+                        throw new NotSupportedException($"tar --no-same-owner -xzf");
```

Comment:
I do not believe --no-same-owner is a standard option for tar across all platforms so we cannot add this in without breaking a lot of customers.